contracts/pcv/IReserveStabilizer.sol

```diff
@@ -0,0 +1,35 @@
+pragma solidity ^0.6.2;
+
+/// @title a Reserve Stabilizer interface
+/// @author Fei Protocol
+interface IReserveStabilizer {
+
+    // ----------- Events -----------
+    event FeiExchange(address indexed to, uint256 feiAmountIn, uint256 amountOut);
+
+    event UsdPerFeiRateUpdate(uint256 basisPoints);
+
+    event Withdrawal(
+        address indexed _caller,
+        address indexed _to,
+        uint256 _amount
+    );
+
+    // ----------- State changing api -----------
+
+    function exchangeFeiForEth(uint256 feiAmount) external returns (uint256);
```

Comment:
in the interface you probably want more generic names (a `DaiReserverStabilizer` with an `exchangeFeiForEth` function would be weird)

Reply:
renamed it to exchangeFei